### PR TITLE
Display admin and channel op in channels tree

### PR DESCRIPTION
**Pull request by Oreonan**

### CODE REVIEW

File: Client/qtTeamTalk/channelstree.cpp [hunk 1]
```diff
@@ -969,6 +969,12 @@ void ChannelsTree::slotUpdateTreeWidgetItem(QTreeWidgetItem* item)
         if (user.nStatusMode & STATUSMODE_STREAM_MEDIAFILE)
             itemtext += tr(" (Streaming media file)");
 
+        if(user.uUserType & USERTYPE_ADMIN)
+            itemtext += tr(" [Administrator]");
```
Comment (bear101) [inline Client/qtTeamTalk/channelstree.cpp:973]: Why brackets instead of parenthesis?

**Comment by Oreonan** [comment]
Because I've wanted to separate informations on status and on rights. 
But if you prefere I can use parenthesis for all?

File: Client/qtTeamTalk/channelstree.cpp [hunk 2]
```diff
@@ -969,6 +969,12 @@ void ChannelsTree::slotUpdateTreeWidgetItem(QTreeWidgetItem* item)
         if (user.nStatusMode & STATUSMODE_STREAM_MEDIAFILE)
             itemtext += tr(" (Streaming media file)");
 
+        if(user.uUserType & USERTYPE_ADMIN)
+            itemtext += tr(" (Administrator)");
```
Comment (bear101) [inline Client/qtTeamTalk/channelstree.cpp:973]: Shouldn't this also be comma separated?

Comment (bear101) [reply Client/qtTeamTalk/channelstree.cpp:973]: Ah, just read your comment. Ok

**Pull request merged**